docs/index.asciidoc

```diff
@@ -76,6 +89,10 @@ A client that receives an HTTP request timeout is expected to retry the entire r
 When this plugin is blocked, it will reject _new_ requests with HTTP `429 Too Many Requests`.
 Clients that receive `429`-s are expected to wait a moment before retrying the request — exponential back-off is encouraged to ease flood recovery.
 
+//
+// END: Elastic-internal implementation details
+////
+
 [id="plugins-{type}s-{plugin}-security"]
 ==== Security
 
```

Comment:
I am okay in keeping the SSL Client Authentication section, but I think it'd be good to add a note that ESF does not actually support presenting client certificates